persistent/picklecache.py

```diff
@@ -106,32 +187,26 @@ def get(self, oid, default=None):
     def mru(self, oid):
         """ See IPickleCache.
         """
-        node = self.ring.next
-        while node is not self.ring and node.object._p_oid != oid:
-            node = node.next
-        if node is self.ring:
-            value = self.data[oid]
-            if value._p_state != GHOST:
```

Comment:
Indirectly, yes. The benchmark numbers are above. PyPy is much faster with the CFFI IRing implementation than it was with the inlined deque-based ring (an intermediate step), which was much faster than the hand-implemented inlined ring. I suspect the same will hold true for CPython, whether or not it uses the deque implementation or the CFFI implementation, but I didn't test that, because all supported CPython versions use the cPickleCache implementation anyway.